public/js/apps/management/main.js

```diff
@@ -1,9 +1,3 @@
-import $ from 'jquery';
 import { init as initApp } from './app';
 
-// Common ajax settings.
-$.ajaxSetup({
-  dataType: 'json',
-});
-
 initApp();
```

Comment:
If this is all this is doing I wonder if we should just kill the main.js files and make app.jsx the entry point for webpack.